recipes/agent_linux.rb

```diff
@@ -46,12 +46,20 @@
     end
   end
 
+  if node['newrelic-infra']['yum']['baseurl'].nil?
```

Comment:
Instead of being checked here, these should just be set in attributes/default.rb